eng/packages/http-client-csharp-mgmt/generator/Directory.Build.props

```diff
@@ -3,7 +3,7 @@
     <Nullable>enable</Nullable>
     <ImportDefaultReferences>false</ImportDefaultReferences>
     <IsShippingLibrary>false</IsShippingLibrary>
-    <NoWarn>$(NoWarn);CS8002</NoWarn>
+    <NoWarn>$(NoWarn);CS8002;NU1608</NoWarn>
```

Comment:
This is needed, because Azure.Generator.Management has a package reference of Azure.Generator. Azure.Generator has a previous version of Microsoft.TypeSpec.Generator.ClientModel, we are updating Microsoft.TypeSpec.Generator.ClientModel to a higher version in the same PR. They will always conflict.

Reply:
But why does Azure.Generator.Mgmt have a direct dependency on ClientModel? It should get it transitively from Azure.Generator.

Reply:
good point, the direct dependency should be removed

Reply:
hmm, this is not really working via transitive dependency of ClientModel from Azure.Generator.
Because ClientModel from Azure.Generator is via previous version, [CI is failing](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4873348&view=logs&j=f375e37b-537d-55d4-1c20-889b1311b0aa&t=037032bd-5708-583e-e0fb-e577beaa6c95&l=221) due to this.

Reply:
Or, we need to do this in 2 steps, only update ClientModel in this PR for Azure.Generator.
And update ClientModel for Azure.Generator.Management in a following PR.

Reply:
Yes, it would need to be two steps.

Reply:
> Or, we need to do this in 2 steps, only update ClientModel in this PR for Azure.Generator. And update ClientModel for Azure.Generator.Management in a following PR.

And this is not working, either. Because Azure.Generator.Management.Test has a project reference of `Azure.Generator.Tests.Common`, which has a package reference of `Microsoft.TypeSpec.Generator.Input` and this is upgrading to a higher version to break the contract.

Reply:
override the MTG.Input package version to previous version in Management.Test csproj, and remove this during the upgrade of Management.